CMSgov/ato-blueprint
# ISPGBSS-666/668: Aggregate CSV export controls and use control label

[STORY: Component narratives for same control need to be merged into a single row](https://jiraent.cms.gov/browse/ISPGBSS-666)
[STORY: Control IDs in CSV export should use leading zeros](https://jiraent.cms.gov/browse/ISPGBSS-668)

Steps to test:

1. Log into Blueprint
2. Go to a Project page
3. Click on the *System Security Plan* button on the right
4. If you need to generate the SSP, click *Next*
5. Click on the *CFACTS (CSV)* link
6. Open the CSV
7. The _Control ID_ column should show the IDs in uppercase and with leading zeros. For example; AU-06(01) rather than au-6.1
8. If there are more than one Component associated with a Control (try AC-02(02)) both narratives should be in the same row in the Private column in the CSV.
